packages/lesswrong/lib/collections/tags/views.ts

```diff
@@ -13,12 +14,13 @@ declare global {
   }
 }
 
-Tags.addDefaultView((terms: TagsViewTerms) => {
+Tags.addDefaultView((terms: TagsViewTerms, _, context?: ResolverContext) => {
+  const currentUser = context?.currentUser ?? null;
+
   return {
     selector: {
-      deleted: false,
-      adminOnly: false,
-      wikiOnly: false
+      wikiOnly: false,
+      ...(!userIsAdminOrMod(currentUser) ? { deleted: false, adminOnly: false } : {}),
     },
```

Comment:
Previously you couldn't see soft-deleted tags even if you were an admin, I changed that because (e.g. in my case) it might be useful to look at the old tag to copy over some of the description